x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -254,7 +255,20 @@ export const SearchBar: FC<SearchBarProps> = ({
 
   const clearField = () => setSearchValue('');
 
-  const emptyMessage = <PopoverPlaceholder darkMode={darkMode} basePath={basePathUrl} />;
+  const noMatchesMessage = <PopoverPlaceholder darkMode={darkMode} basePath={basePathUrl} />;
+  const emptyMessage = (
+    <div
```

Comment:
NIT: I think we could use `EuiFlexGroup` or `EuiFlexItem` here instead of manually specifying the flex css on a raw div.